src/ICal/ICal.php

```diff
@@ -688,7 +688,11 @@ public function iCalDateWithTimeZone(array $event, $key, $format = self::DATE_TI
 
         // Force time zone
         if (isset($dateArray[0]['TZID'])) {
-            $dateTime->setTimezone(new \DateTimeZone($dateArray[0]['TZID']));
+            if($this->isValidTimeZoneId($dateArray[0]['TZID'])){
+                $dateTime->setTimezone(new \DateTimeZone($dateArray[0]['TZID']));
+            }else{
+                $dateTime->setTimezone($this->defaultTimeZone);
```

Comment:
Assume this should be `$dateTime->setTimezone(ew \DateTimeZone($this->defaultTimeZone));`